Circle.js

```diff
@@ -203,7 +204,7 @@ export class ProgressCircle extends Component {
               ]}
               allowFontScaling={allowFontScaling}
             >
-              {formatText(progressValue)}
+              {progress ? formatText(progress._value) : this.forceUpdate()}
```

Comment:
Why is this needed? Calling `this.forceUpdate()` from the render function might cause an infinite loop.